src/shared/components/data-table/DataTable.scss

```diff
@@ -0,0 +1,54 @@
+@import 'src/styles/common';
+
+.wrapper {
+  --table-background: $white;
+  overflow: auto;
+
+  &ThemeDark {
+    background-color: $light-grey;
+  }
+}
+
+.table {
+  width: var(--table-width, 100%);
+  height: var(--table-height, 500px);
+  overflow: auto;
+  display: block;
```

Comment:
- Are the width and height parameters here really necessary? How does a table behave when it has only a couple of rows?
- What's the purpose of specifying `display: block`? I thought a table should have `display: table` (which it will have by default)

Reply:
> Are the width and height parameters here really necessary? How does a table behave when it has only a couple of rows?

I think it is to keep the height fixed to prevent it from jumping when we filter / sort.

> What's the purpose of specifying display: block? I thought a table should have display: table (which it will have by default)

I couldn't fix the height otherwise.

Reply:
> > What's the purpose of specifying display: block? I thought a table should have display: table (which it will have by default)
> 
> I couldn't fix the height otherwise.

Mmm, this is not good. Something to look deeper into / talk through together. We shouldn't be overriding the default display behaviour of a table.

Reply:
> I couldn't fix the height otherwise.

Is this what you mean?

https://user-images.githubusercontent.com/6834224/187908844-d7275a96-9ea2-4fe2-a1ba-6a5083b6da57.mov

Try wrapping the `table` component into its own div that will have a defined height.